distributions/test/src/test/resources/application.yml

```diff
@@ -20,4 +21,4 @@ replication:
   period: 1
   connectionTimeout: 30
   receiveTimeout: 60
-  localSite: source-id
\ No newline at end of file
+  localSite: destination-id
```

Comment:
Sites need to be regional for the Ion Architecture to work with them, and we only harvest from regional sites so the local site will be the destination.